src/Microsoft.TemplateEngine.Cli/CommandParsing/SupportedFilterOptions.cs

```diff
@@ -0,0 +1,81 @@
+using Microsoft.TemplateEngine.Cli.TemplateResolution;
+using Microsoft.TemplateEngine.Edge.Template;
+using System.Collections.Generic;
+
+namespace Microsoft.TemplateEngine.Cli.CommandParsing
+{
+    internal static class SupportedFilterOptions
+    {
+
+        internal static readonly IReadOnlyCollection<FilterOption> SupportedListFilters = new List<FilterOption>()
+        {
+            SupportedFilterOptions.AuthorFilter,
+            SupportedFilterOptions.BaselineFilter,
+            SupportedFilterOptions.LanguageFilter,
+            SupportedFilterOptions.TypeFilter
+        };
+
+        internal static FilterOption BaselineFilter
+        {
+            get
+            {
+                return new TemplateFilterOption()
```

Comment:
Would it make sense to have only one instance of each filter as a static field in the class and return that same instance whenever requested? We are creating a new instance each time the property is called and each instance seems to be identical.